pkg/instrumentation/podmutator.go

```diff
@@ -402,14 +404,42 @@ func (pm *instPodMutator) selectInstrumentationInstanceFromNamespace(ctx context
 		if err != nil {
 			pm.Logger.Error(err, "unable to retrieve cloudwatch agent config for instrumentation")
 		}
-		return getDefaultInstrumentation(config)
+		return getDefaultInstrumentation(config, extraJavaEnvVar)
 	case s > 1:
 		return nil, errMultipleInstancesPossible
 	default:
 		return &otelInsts.Items[0], nil
 	}
 }
 
+func getJavaEnvVar(ns corev1.Namespace, pod corev1.Pod) map[string]string {
+	targetSystems := getTargetSystems(ns, pod, annotationJMXJVM, "")
+	targetSystems = getTargetSystems(ns, pod, annotationJMXTomcat, targetSystems)
+	targetSystems = getTargetSystems(ns, pod, annotationJMXKafka, targetSystems)
+	targetSystems = getTargetSystems(ns, pod, annotationJMXKafkaConsumer, targetSystems)
+	targetSystems = getTargetSystems(ns, pod, annotationJMXKafkaProducer, targetSystems)
+	if targetSystems != "" {
+		return map[string]string{
+			"OTEL_EXPORTER_OTLP_METRICS_ENDPOINT":            "http://cloudwatch-agent.amazon-cloudwatch:4314/v1/metrics",
+			"OTEL_INSTRUMENTATION_RUNTIME_TELEMETRY_ENABLED": "false",
+			"OTEL_JMX_ENABLED":                               "true",
+			"OTEL_JMX_TARGET_SYSTEM":                         targetSystems,
+		}
+	}
+	return make(map[string]string)
```

Comment:
No point in creating an empty map.
```suggestion
	return nil
```

Reply:
I disagree. When we use nil we could have panics. I like the idea of empty objects over nulls.

Reply:
What are we doing with the map that would result in panics? It's pretty standard in go to pass around nil maps. Unless we're writing to it, it shouldn't panic.

https://go.dev/ref/spec#Map_types
> A nil map is equivalent to an empty map except that no elements may be added.

Reply:
Guess I'm too java focused. Will change